code/game/gamemodes/cult/runes.dm

```diff
@@ -803,7 +812,15 @@ var/list/sacrificed = list()
 /obj/effect/rune/proc/wall(var/mob/living/user)
 	user.say("Khari[pick("'","`")]d! Eske'te tannin!")
 	src.density = !src.density
-	user.take_organ_damage(2, 0)
+	user.rune_apply_damage(10, 15)
+	if(src.density)
+		user << "<span class='danger'>Your blood flows into the rune, and you feel that the very space over the rune thickens.</span>"
+	else
+		user << "<span class='danger'>Your blood flows into the rune, and you feel as the rune releases its grasp on space.</span>"
+	return

```

Comment:
what.

The `return` here stops everything on from this line from executing. Why?